README.md

```diff
@@ -62,7 +62,7 @@ The most important parameter of this command is `--ani_thresh`: this is average
 ### Run the YACHT algorithm
 After this, you are ready to perform the hypothesis test for each organism in your reference database. This can be accomplished with something like:
 ```bash
-python run_YACHT.py --ref_matrix 'gtdb_ani_thresh_0.95_ref_matrix_processed.npz' --ksize 31 --sample_file 'sample.sig.zip' --ani_thresh 0.95 --significance 0.99 --min_coverage 1 --outfile 'yacht_results.csv'
+python run_YACHT.py --json 'gtdb_ani_thresh_0.95_config.json' --ksize 31 --sample_file 'sample.sig.zip' --ani_thresh 0.95 --significance 0.99 --min_coverage 1 --outdir './'
```

Comment:
Getting an error here:

YACHT % python run_YACHT.py --json 'gtdb_ani_thresh_0.95_config.json' --ksize 31 --sample_file 'sample.sig.zip' --ani_thresh 0.95 --significance 0.99 --min_coverage 1 --outdir './'
usage: run_YACHT.py [-h] --json JSON --sample_file SAMPLE_FILE
                    [--significance SIGNIFICANCE] [--keep_raw]
                    [--show_present_only]
                    [--min_coverage MIN_COVERAGE [MIN_COVERAGE ...]]
                    [--out_filename OUT_FILENAME] --outdir OUTDIR
run_YACHT.py: error: unrecognized arguments: --ksize 31 --ani_thresh 0.95

Reply:
Sorry.  These parameters `--ksize 31 --ani_thresh 0.95` are invalid now. We should remove that.